WordPress/Classes/ViewRelated/Tools/Views/TimeZoneSearchBar.swift

```diff
@@ -0,0 +1,62 @@
+import UIKit
+
+final class TimeZoneSearchBar: UIView {
```

Comment:
Sorry if that's confusing; when I was doing going through the code test part of the process, I had seen that in WooCommerce iOS almost everything is marked final where it can be so I had presumed that this was a best practice here!  In terms of other reasoning, there is a small performance benefit also mentioned in Apple's Swift blog [here](https://developer.apple.com/swift/blog/?id=27).  I was trying to be consistent with the codebase / org so maybe I applied a standard from a different codebase incorrectly.  I've removed final 😄

Reply:
I think the reasoning is pretty solid, so unless we foresee subclassing, I'd say we can re-instate the final

Reply:
Done! thx